admin/class-admin-apple-preview.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * This class is in charge of handling Apple News previews
+ */
+class Admin_Apple_Preview extends Apple_News {
+	/**
+	 * Constructor.
+	 */
+	function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'register_assets' ) );
+	}
+
+	/**
+	 * Register assets for the options page.
+	 *
+	 * @param string $hook
+	 * @access public
+	 */
+	public function register_assets( $hook ) {
+		if ( ! in_array( $hook, array(
+			'apple-news_page_apple-news-options',
+			'admin_page_apple-news-theme-preview'
+		) ) ) {
+			return;
+		}
+
+		wp_enqueue_style( 'apple-news-preview-css', plugin_dir_url( __FILE__ ) .
```

Comment:
This is a convention across the plugin, so maybe we'll clean it up in a future version.